src/test/java/seedu/address/logic/commands/ListScoreCommandTest.java

```diff
@@ -0,0 +1,465 @@
+    @Test
+    public void execute_nullModel_throwsNullPointerException() {
+        assertThrows(NullPointerException.class, (
+            ) -> new ListScoreCommand(VALID_TUTORIAL_NAME_1, VALID_ASSESSMENT_NAME_1).execute(null));
+    }
+
+    @Test
+    public void equals() {
+        ListScoreCommand firstCommand = new ListScoreCommand(VALID_TUTORIAL_NAME_1, VALID_ASSESSMENT_NAME_1);
+        ListScoreCommand secondCommand = new ListScoreCommand(VALID_TUTORIAL_NAME_1, VALID_ASSESSMENT_NAME_2);
+        ListScoreCommand thirdCommand = new ListScoreCommand(VALID_TUTORIAL_NAME_2, VALID_ASSESSMENT_NAME_1);
+        ListScoreCommand fourthCommand = new ListScoreCommand(VALID_TUTORIAL_NAME_2, VALID_ASSESSMENT_NAME_2);
+
+        // same object -> returns true
+        assertTrue(firstCommand.equals(firstCommand));
+        assertTrue(thirdCommand.equals(thirdCommand));
+
+        // same values -> returns true
+        ListScoreCommand firstCommandCopy = new ListScoreCommand(VALID_TUTORIAL_NAME_1, VALID_ASSESSMENT_NAME_1);
+        assertTrue(firstCommand.equals(firstCommandCopy));
+        ListScoreCommand thirdCommandCopy = new ListScoreCommand(VALID_TUTORIAL_NAME_2, VALID_ASSESSMENT_NAME_1);
+        assertTrue(thirdCommand.equals(thirdCommandCopy));
+
+
+        // different types -> returns false
+        assertFalse(firstCommand.equals(1));
+
+        // null -> returns false
+        assertFalse(firstCommand.equals(null));
+
+        // different values -> returns false
+        assertFalse(firstCommand.equals(secondCommand));
+        assertFalse(firstCommand.equals(thirdCommand));
+        assertFalse(firstCommand.equals(fourthCommand));
+    }
+
+    @Test
+    public void execute_validInput_commandSuccess() throws CommandException {
+        String expectedMessage =
+                String.format(ListScoreCommand.MESSAGE_SUCCESS, VALID_TUTORIAL_NAME_1,
+                        VALID_ASSESSMENT_NAME_1, VALID_ASSESSMENT_1.getFullMark());
+        ListScoreCommand command = new ListScoreCommand(VALID_TUTORIAL_NAME_1, VALID_ASSESSMENT_NAME_1);
+        ModelStubListScore model = new ModelStubListScore();
+        CommandResult result = command.execute(model);
+        assertEquals(expectedMessage, result.getFeedbackToUser());
+    }
+
+    @Test
+    public void execute_invalidTutNameInput_commandFailure() {
+        String expectedMessage = Messages.MESSAGE_TUTORIAL_NOT_FOUND;
+        ListScoreCommand command = new ListScoreCommand(VALID_TUTORIAL_NAME_2, VALID_ASSESSMENT_NAME_1);
+        try {
+            command.execute(new ModelStubListScore());
+        } catch (CommandException ce) {
+            assertEquals(expectedMessage, ce.getMessage());
+        }
+    }
+
+    @Test
+    public void execute_invalidAssessmentNameInput_commandFailure() {
+        String expectedMessage = Messages.MESSAGE_ASSESSMENT_NOT_FOUND;
+        ListScoreCommand command = new ListScoreCommand(VALID_TUTORIAL_NAME_1, VALID_ASSESSMENT_NAME_2);
+        try {
```

Comment:
maybe you can use assertThrows for consistency with other test classes?

Reply:
sounds good. Thank you!